### ISSUE TITLE
Insufficient alt text (high)

### ISSUE THREAD

**Issue by MelanieHotchkiss** [magda-io/magda#1911]
### Problem description
Intopia issue 8 - Feature images all have the same alt text of “title”.

### Problem reproduction steps
As each image is linked to an external website or internal webpage, the alt text should usually describe the link destination. In this case, however, each linked image is adjacent to a text link with the same destination. To avoid repeating the link destinations, the image and the text should be wrapped in a single link and the image alt text should have a null value.

### Screenshot / Design / File reference
[8] Insufficient alt text
Type: WCAG 2.1 failure
Fails 1.1.1 Non-text Content - Level A
Fails 2.4.4 Link Purpose (In Context) - Level A
Location: Feature image, home page
Platform: All
Severity: High
What’s the issue?
Feature images all have the same alt text of “title”.
Why it matters NVDA/FF announces “link graphic title”. JAWS/IE announces “link graphic homepage/story4” which is the filename. Talkback/Android announces “title link”. VoiceOver/Safari announces “Link. Image. Title image.” None of this gives the user sufficient information about the image or the link destination.
Our recommendation
As each image is linked to an external website or internal webpage, the alt text should usually describe the link destination. In this case, however, each linked image is adjacent to a text link with the same destination. To avoid repeating the link destinations, the image and the text should be wrapped in a single link and the image alt text should have a null value.
Code sample:
<a href="https://ipnova.ipaustralia.gov.au/#/" >
<img class="story-t
<h2 class="story-title">Intellectual property in Australia: a visualisation</h2>
</a>
